com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -823,15 +863,22 @@ var Module = {
             Module._waitingForArchive = true;
         } else {
             Module.preloadAll();
-            Progress.removeProgress();
-            if (Module.callMain === undefined) {
-                Module.noInitialRun = false;
-            } else {
-                Module.callMain(Module.arguments);
+            if (Module._isEngineLoaded) {
```

Comment:
Check if engine binary loaded and JS runtime is initialize (it means that 'onRuntimeInitialized' was called).